libnmstate/netapplier.py

```
@@ -42,9 +41,10 @@
 VERIFY_RETRY_TIMEOUT = 5
 
 
-@_warn_keyword_as_positional
```

I prefer raise an exception instead of silently ignore arguments.

---

So, instead of *, should I create a decorator that checks for positional args and raises an exception?

---

I think there is a misunderstanding. @cathay4t this will not ignore any arguments. If extra positional arguments are provided, Python will raise a TypeError. The tests check this.

---

what should I do?

---

For this we wait for @cathay4t feedback, but you also need to address the comment about pylint and rebase to resolve a conflict.

---

@tyll I already addressed the pylint issue. I was waiting to push both together.

---

My fault. The `TypeError` is enough for me.